packages/server/src/sdk/app/applications/sync.ts

```diff
@@ -107,9 +100,11 @@ export async function syncUsersToAllApps(userIds: string[]) {
   }
   const resp = await Promise.allSettled(promises)
   const failed = resp.filter(promise => promise.status === "rejected")
-  if (failed.length > 0) {
-    const reasons = failed.map(fail => (fail as PromiseRejectedResult).reason)
-    logging.logAlert("Failed to sync users to apps", reasons)
+  const reasons = failed
+    .map(fail => (fail as PromiseRejectedResult).reason)
+    .filter(reason => !dbCore.is409Error(reason))
+  if (reasons.length > 0) {
+    logging.logWarn("Failed to sync users to apps", reasons)
```

Comment:
Commented above on Martys comment, IMHO this is not a critical alert if anything goes wrong as there are multiple other mechanisms which will correctly update it otherwise. My point with this is that it should not be urgently looked at if an error occurs here.

Reply:
I do agree that's it's not a down tools scenario. I'd just be concerned that if there was a problem, we should be able to detect it rather than relying on user reported problems. I feel the same with most of the log alerts, they are usually very targeted scenarios that we have added in _some_ places. They're not an overall reflection of system health. Really we should be monitoring:
- 50Xs
- Failed jobs

Reply:
I agree - although is this not what the warnings are for? The fact we don't monitor them isn't necessarily a reason that this *isn't* a warning - ideally given the new strategy towards log alerts we should focus on making them a down tools scenario, and then warnings become what alert was before.

This really comes down to how we want to handle this going forward - my reason for down-grading this to a warning was due to the nature of the new log alerts channel, if this does throw some sort of error its highly unlikely to be looked at in any meaningful way as it is *very* unlikely to create a scenario that could cause a problem for a user. Ideally we should be doing some sort of rate monitoring for the warnings, if a specific warning is occurring too often then we can look into why say 1000 sync user failed messages have occurred.

Reply:
> Ideally we should be doing some sort of rate monitoring for the warnings
I think this is the right way to go about it. Unless if affects all customers, it shouldn't be in the alerts channel.

Reply:
I agree, I think I would like to set this strategy for the team:
1. If it happens, you want to know, it could be bad and it needs looked into - log alert
2. If it happens, its fine, it doesn't directly effect anyone, but if 1000 of them happen in an hour, its worth looking into what happened - log warning

Based on this - are we happy to proceed/approve - while we don't have have point 2 setup right now, I would like to make it so that if I see `logAlert` in a PR I can ask if its a critical issue - if it happens would you want to immediately debug it; otherwise put in a log warning.